stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -970,8 +970,23 @@ function complete_keyword_argument(partial, last_idx, context_module)
     end
 
     suggestions = Completion[KeywordArgumentCompletion(kwarg) for kwarg in kwargs]
-    append!(suggestions, complete_symbol(nothing, last_word, Returns(true), context_module))
-    append!(suggestions, complete_keyval(last_word))
+
+    name_suggestions = complete_symbol(nothing, last_word, Returns(true), context_module)
+
+    # Only add these if not in kwarg space. i.e. not in `foo(; `
+    if kwargs_flag == 0
+        append!(suggestions, name_suggestions)
+        append!(suggestions, complete_keyval(last_word))
+    else
+        # if any names match kwargs, offer to complete them without the trailing `=`
+        for s1 in suggestions
+            for s2 in name_suggestions
+                if _completion_text(s1) == _completion_text(s2) * "="
+                    push!(suggestions, s2)
+                end
+            end
+        end
```

Comment:
```suggestion
        if !isempty(kwargs)
            append!(suggestions, s for s in name_suggestions if _completion_text(s) ∈ kwargs)
        end
```

Might as well avoid a double loop since `kwargs` is already the list of keyword argument suggestions (without the trailing "=") in the form of a `Set{String}`.

Reply:
Perfect, thanks